templates/shortcode-receipt.php

```diff
@@ -150,12 +150,12 @@
 						<div class="edd_purchase_receipt_product_name">
 							<?php echo esc_html( $item['name'] ); ?>
 							<?php if( ! is_null( $price_id ) ) : ?>
-							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
+							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo esc_attr( edd_get_price_option_name( $item['id'], $price_id, $payment->ID ) ); ?></span>
 							<?php endif; ?>
 						</div>
 
 						<?php if ( $edd_receipt_args['notes'] ) : ?>
-							<div class="edd_purchase_receipt_product_notes"><?php echo wpautop( edd_get_product_notes( $item['id'] ) ); ?></div>
+							<div class="edd_purchase_receipt_product_notes"><?php echo wp_kses_post(wpautop( edd_get_product_notes( $item['id'] ) ) ); ?></div>
```

Comment:
@christ0ph3r Ah yes, my bad on the semi-colon. So `strip_tags` requires the second parameter be a string.

Pretty sure you *can* use `wp_kses_post()` for escaping output: https://vip.wordpress.com/documentation/best-practices/security/validating-sanitizing-escaping/#escaping-securing-output

https://developer.wordpress.org/plugins/security/securing-output/

Though there are possible performance concerns with it. I'll let @pippinsplugins or @cklosowski chime in on the best way to handle this.